.gitignore

```diff
@@ -130,3 +130,4 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+.vscode/
```

Comment:
nit: I kind of like the way the .gitignore is commented with what a lot of this ignored stuff is associated with. I would recommend a small change.

```suggestion

# VSCode IDE
.vscode/
```